depop-app/src/components/ProductCard.js

```diff
@@ -0,0 +1,21 @@
+import "./ProductCard.css"
+
+export const ProductCard = ({product}) => {
+    console.log(product);
```

Comment:
Remove?

Reply:
I was thinking of keeping it in, so candidates could quickly see the response from the endpoint - but I guess then can just call the API in their browser.